tests/test_data/survey/single_survey_eq.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It's minor, but you have BRES long name Bricks short and it is a eQ when both are SEFT